packages/ts/src/components/timeline/index.ts

```diff
@@ -232,7 +232,7 @@ export class Timeline<Datum> extends XYComponentCore<Datum, TimelineConfig<Datum
         .attr('y', yStart + y + (config.rowHeight - height) / 2)
         .attr('width', config.showEmptySegments
           ? Math.max(config.lineCap ? height : 1, width)
-          : width)
+          : Math.max(0, width))
```

Comment:
@reb-dev I'm thinking that if the width is negative most likely it's due to misconfiguration of the component. So it's worth adding a console warning to give users a hint. 

We usually use the `Unovis | <Component>:` prefix for warning messages:
```ts
console.warn('Unovis | Timeline: ...')
```

Reply:
@rokotyan yes, you're right! I just added that.